test/MessageTest.php

```diff
@@ -790,4 +790,29 @@ public function testHeaderUnfoldingWorksAsExpectedForMultipartMessages()
         $this->assertContains('multipart/alternative', $contentType->getFieldValue());
         $this->assertContains($multipartContent->getMime()->boundary(), $contentType->getFieldValue());
     }
+
+    /**
+     * @group 19
+     */
+    public function testCanParseMultipartReport()
+    {
+        $raw = file_get_contents(__DIR__ . '/_files/zend-mail-19.txt');
+        $message = Message::fromString($raw);
```

Comment:
The exception is being thrown by Headers::fromString(), not Message::fromString().